.travis.yml

```diff
@@ -6,24 +6,24 @@ rvm:
   - jruby-9.2.0.0
   - jruby-9.2.7.0
 jdk:
-  - oraclejdk8
+  - oraclejdk9
```

Comment:
hmm I wouldn't use java 9

doesn't `openjdk8` work?

Reply:
Travis' JDK install script seems to be broken for some time now... during installation we got this output:

```
Installing oraclejdk8
$ export JAVA_HOME=~/oraclejdk8
$ export PATH="$JAVA_HOME/bin:$PATH"
$ ~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"
install-jdk.sh 2019-05-02
Expected feature release number in range of 9 to 13, but got: 8
The command "~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"" failed and exited with 3 during .
```

This will be solved by merging #39 though.